store.html

```diff
@@ -168,6 +168,25 @@
         <td></td>
         <td></td>
       </tr>
+      <tr>
+        <td>Totals Per Hour</td>
+        <td></td>
```

Comment:
You're going to want to generate this `<tr>` and insert it after the store `<tr>`s.